src/connect.ts

```diff
@@ -0,0 +1,103 @@
+import { Coin, DirectSecp256k1HdWallet } from "@cosmjs/proto-signing";
+import { Block, calculateFee, GasPrice } from "@cosmjs/stargate";
+import {SigningStargateClient, StargateClient } from "@cosmjs/stargate";
+import { ArchwayClient, ArchwayTxFilter, ArchwayTXSearch } from "./query";
+import { SigningCosmWasmClient, CosmWasmClient } from "@cosmjs/cosmwasm-stargate";
+import {uploadContract} from "./wasm"
+// Wrapper for connecting to RPC client from COSM JS
+// Can either connect to testnet or mainnet
+// TODO: add mainnet
+
+
+// Interface that holds the Signing Client and Wallet Account
+export class ArchwaySigningClient {
```

Comment:
I am not fully following this client.
Maybe @drewstaylor and @aelesbao can jump something I might have missed.

We are building this to be hardcoded Archway client?
Can't we provide everything as parameters. Sure, we can have constants or config files pre-loaded for our own networks

But the user should always be able to set to another cosmos network (IMO)... or even to it's own local environment